server/routes/authorisations/projects.js

```diff
@@ -32,7 +32,7 @@ module.exports.define = (apiPrefix) => {
 
 module.exports.isAllowed = (req, res, next) => {
   if (req.params.id) {
-    if (req.app.locals.project && req.app.locals.project.isOwner(req.user.userId)) {
+    if ((req.app.locals.project && req.app.locals.project.isOwner(req.user.userId)) || req.user.role === 'admin') {
```

Comment:
I'm not really sure this is the best way to handle this. Open to suggestions!

Reply:
i guess that to avoid that, you could call utils.isAllowed;
In the current scenario, the roles are ignored; By calling isAllowed, you would check the user role as well

Reply:
Wouldn't `util.isAllowed` return true for a `'basic'` role as well then, allowing any user to see any project? `util.isAllowed` only does role-based checking, not target-based, and `'basic'` is a role allowed to use this endpoint too.

Reply:
Inside the truthy block means you already checked the ownership (as a replacement of the next, not as part of the condition).

Reply:
But admins don't have ownership, so you wouldn't get into the truthy block.